app/models/manageiq/providers/container_manager.rb

```diff
@@ -19,6 +19,7 @@ class ContainerManager < BaseManager
     has_many :container_build_pods, :foreign_key => :ems_id, :dependent => :destroy
     has_many :container_templates, :foreign_key => :ems_id, :dependent => :destroy
     has_one :container_deployment, :foreign_key => :deployed_ems_id, :inverse_of => :deployed_ems
+    has_many :miq_alert_statuses, :through => :container_nodes
```

Comment:
@moolitayer I don't think this is optimal as we grow in number of entities supporting alerts. You should probably take all the `MiqAlertStatuses` that are related to this provider (not just the nodes ones).